lib/actions/addLinkToProjectAction.js

```diff
@@ -12,21 +13,21 @@ class AddLinkToProjectAction {
     }
     
     let linkParams = { 
-      label: parsedUserInput[0],
-      link: parsedUserInput[1]
+      label: parsedUserInput[1],
+      link: parsedUserInput[2]
     }
-      
     return linkParams
   }
 
-  async execute(params){ 
+  async execute(params, currentProject){ 
```

Comment:
Instead of `currentProject` which is needed, our [diagram](https://www.draw.io/#G14szyOL_wrnmm5Do7af_PeonDV9R114Wc) suggests we use `context` with properties for `project`. This allows our `execute` method context to grow or be open for extensibility, while being closed to modification.

Something like this:

```
{
  project: project
}
```